src/Myriad.Core/AstExtensions.fs

```diff
@@ -0,0 +1,495 @@
+namespace Myriad.Core
+
+open System
+open FSharp.Compiler.Syntax
+open FSharp.Compiler.Text.Range
+open FSharp.Compiler.Text.Position
+open FSharp.Compiler.Xml
+open FSharp.Compiler.SyntaxTrivia
+
+[<AutoOpen>]
+module AstExtensions =
```

Comment:
The extensions in here are super nice to have! Really cool to see such a rich set available.